networks/base/indexing.csv

```diff
@@ -0,0 +1,19 @@
+slug,provider,chain,address,technology,plan,trial,accessPrice,queryPrice,uptimeSla,bandwidthSla,blocksBehindSla,supportSla,starred,availableApis,limitations,securityImprovements,monitoringAndAnalytics,regions,additionalFeatures,verifiedUptime,verifiedLatency,verifiedBlocksBehindAvg,actionButtons
+thegraph-mainnet,The Graph,mainnet,null,Subgraph Indexing,Subscription-based,FALSE,0,GRT Fee,null,null,null,null,TRUE,"[""GraphQL""]","[""Rate limits apply""]","[""DDoS Protection"",""IP Protection""]","[""Subgraph logs"",""Usage analytics""]","[""Global""]","[""Decentralized Subgraphs"", ""Custom Queries""]",null,null,null,"[""[Buy](https://thegraph.com)"", ""[Docs](https://thegraph.com/networks/base/?subnetwork=base-mainnet)""]"
+thegraph-testnet,The Graph,sepolia,null,Subgraph Indexing,Free,FALSE,0,0,null,null,null,null,FALSE,"[""GraphQL""]","[""Rate limits apply""]","[""DDoS Protection"",""IP Protection""]","[""Subgraph logs"",""Usage analytics""]","[""Global""]","[""Decentralized Subgraphs"", ""Custom Queries""]",null,null,null,"[""[Docs](https://thegraph.com/networks/base/?subnetwork=base-sepolia)""]"
+chainstack-mainnet-developer,Chainstack,mainnet,null,Subgraph Indexing,Developer,FALSE,0,0,null,null,null,null,FALSE,"[""GraphQL""]","[""3,000,000 request units per month, 20 request units per subgraph request"", ""Up to 1000 RPS"", ""1 subgraph""]","[""DDoS Protection"",""IP Protection""]","[""Subgraph logs"",""Usage analytics""]","[""Global""]","[""Custom Subgraphs"",""GraphQL Queries"",""Community Support""]",null,null,null,"[""[Docs](https://docs.chainstack.com/docs/chainstack-subgraphs-tutorials)""]"
```

Comment:
@nkorn1lov what's this "developer" thingy?
I thought there are no free subgraphs on Chainstack